vac/32/rln-v1.md

```diff
@@ -321,15 +355,13 @@ which together with the ZK-SNARK circuit enable the above mentioned features.
 | **External nullifier**    | Poseidon hash of [Epoch, RLN Identifier]. An identifier that groups signals and can be thought of as a voting booth.  |
 | **Internal nullifier**    | Poseidon hash of [A1]. This field ensures that a user can send only one valid signal per external nullifier without risking being slashed. Public input of the circuit.   |
 
-
-
-### ZK Circuits specification
+#### ZK Circuits specification
 
 Anonymous signaling with a controlled rate limit is enabled by proving that the user is part of a group which has high barriers to entry (form of stake) and
 enabling secret reveal if more than 1 unique signal is produced per external nullifier.
-The membership part is implemented using membership [Merkle trees](https://en.wikipedia.org/wiki/Merkle_tree) and Merkle proofs,
-while the secret reveal part is enabled by using the Shamir's Secret Sharing scheme.
-Essentially the protocol requires the users to generate zero-knowledge proof to be able to send signals and participate in the application.
+The membership part is implemented using membership [Merkle trees](https://en.wikipedia.org/wiki/Merkle_tree) and Merkle proofs, while the secret reveal part is enabled by using the Shamir's Secret Sharing scheme.
+Essentially the protocol requires the users to generate zero-knowledge proof to be able to send signals and 
```

Comment:
there are usages of `zero-knowledge proof` and `zk-proof` interchangeably, either we should mention this up top, or use one throughout

Reply:
I replaced `zk-proof` with `zero-knowledge proof` within the spec. Expect for `zk_proof` value, I'm not sure if this should be changed.